cli/docker/container_manager.go

```diff
@@ -0,0 +1,545 @@
+package docker
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/bitrise-io/bitrise/log"
+	"github.com/bitrise-io/bitrise/models"
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/v2/redactwriter"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/client"
+)
+
+type RunningContainer struct {
+	ID   string
+	Name string
+}
+
+type containerCreateOptions struct {
+	name       string
+	volumes    []string
+	command    string
+	workingDir string
+	user       string
+}
+
+func (rc *RunningContainer) Destroy() error {
+	_, err := command.New("docker", "rm", "--force", rc.Name).RunAndReturnTrimmedCombinedOutput()
```

Comment:
If the output is swallowed and only the error is printed, then it's going to show something like `exit status 2`.

Reply:
Also, did you look into the `--volumes` flag of `rm`? We should probably clean up the anonymous volumes too.